### PR TITLE
Feature Icon.createFromIconfontCN adaptive http or https

### PR DESCRIPTION
**Pull request by wizard-a**
issues  #12316 

** Check **

* [ ] Make sure that you propose PR to right branch: bugfix for `master`, feature for branch `feature`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

**NewFeature**

  * [x] Update API docs for the component.
  * [x] Add unit tests for the feature.

### CODE REVIEW

**Comment by afc163** [comment]
image

**Comment by HeskeyBaozi** [comment]
Adding some detail descriptions in the document `scriptUrl` will be better 👍

**Comment by HeskeyBaozi** [comment]
I cannot rebase and merge this branch for some conflicts.

File: components/icon/index.zh-CN.md [hunk 1]
```diff
@@ -135,3 +135,43 @@ ReactDOM.render(
...
+`options` 的配置项如下：
+
+| 参数 | 说明 | 类型 | 默认值 |
+| --- | --- | --- | --- |
+| scriptUrl | [iconfont.cn](http://iconfont.cn/) 项目在线生成的 `js` 地址，在 `namespace` 也设置的情况下有效 | string | - |
+| extraCommonProps | 给所有的 `svg` 图标 `<Icon />` 组件设置额外的属性 | `{ [key: string]: any }` | {} |
+
+在 `namespace` 和 `scriptUrl` 都设置有效的情况下，组件在渲染前会自动引入 [iconfont.cn](http://iconfont.cn/) 项目中的图标符号集，无需手动引入。
+
+见 [iconfont.cn 使用帮助](http://iconfont.cn/help/detail?spm=a313x.7781069.1998910419.15&helptype=code) 查看如何生成 `js` 地址。
```
[the remark this answers is not in the document]
Comment (jiechud) [reply components/icon/index.zh-CN.md]: 不好意思，不够细心，以后会注意，已经重新提交修改

File: components/icon/index.zh-CN.md [hunk 2]
```diff
@@ -135,3 +135,15 @@ ReactDOM.render(
...
+
+`Icon` 中的 `component` 组件的属性如下：
+
+| 字段 | 说明 | 类型 | 只读值 |
+| --- | --- | --- | --- |
+| width | `svg` 元素宽度 | string \| number | '1em' |
+| height | `svg` 元素高度 | string \| number | '1em' |
+| fill | `svg` 元素填充的颜色 | string | 'currentColor' |
+| className | 计算后的 `svg` 类名 | string | - |
+| style | 计算后的 `svg` 元素样式 | CSSProperties | - |
```
Comment (afc163) [inline components/icon/index.zh-CN.md]: 这段应该也是重复的。

File: components/icon/__tests__/index.test.js [hunk 3]
```diff
@@ -199,4 +199,14 @@ describe('utils', () => {
         'home-o-fill', 'home-fill-o', 'home-o-twotone', 'home-o']
     );
   });
+
+  it('handleSciprtUrl() should workf', () => {
```
Comment (afc163) [inline components/icon/__tests__/index.test.js]: workf => work

**Comment by jiechud** [comment]
@afc163  ci 出问题了，我看是table 问题，我没有修改table, 我要怎么处理呢

**Comment by afc163** [comment]
试一下 rebase feature

**Comment by jiechud** [comment]
@yesmeck 内网部署，没有用https，岂不是不能用了

**Pull request merged**